src/templates/login/login.tsx

```diff
@@ -40,18 +56,58 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         defaultMessage: "inloggen",
       });
 
+  const _titleLogin = titleLogin
+    ? titleLogin
+    : intl.formatMessage({
+        id: "mykn.templates.Login.titleLogin",
+        description: "templates.Login: The login title",
+        defaultMessage: "Welkom Terug",
+      });
+
```

Comment:
`labelForgotPassword` and I think one label (for both the title and link) should be sufficient, given that both the fallback and React Intl implementation support placeholders for values.